move install instructions to docs\installing.md (#1372)
diasurgical/devilutionX: README.md

For a full list of changes see our [changelog](docs/CHANGELOG.md).

# How To Install:

> Download the latest DevilutionX release and extract the contents to the location or your choice, or [build from source](#building-from-source)

Reviewer: ```suggestion
Download the latest [DevilutionX release](https://github.com/diasurgical/devilutionX/releases) and extract the contents to a location of your choosing.
```
